README.md

```diff
@@ -16,42 +18,177 @@ or a given CSV in the same format with the
 [cisagov/hash-http-content](https://github.com/cisagov/hash-http-content)
 Python library. Then it will output CSVs with agency and domain level results.
 
-## Usage ##
+## Running ##
 
-### Install ###
+### Running with Docker ###
 
-Pull `cisagov/vdp-scanner` from the Docker repository:
+To run the `cisagov/vdp-scanner` image via Docker:
 
 ```console
-docker pull cisagov/vdp-scanner
+docker run cisagov/vdp-scanner:0.0.2
 ```
 
-Or build `cisagov/vdp-scanner` from source:
+### Running with Docker Compose ###
 
-```console
-git clone https://github.com/cisagov/vdp-scanner-docker.git
-cd vdp-scanner-docker
-docker-compose build
-```
+1. Create a `docker-compose.yml` file similar to the one below to use [Docker Compose](https://docs.docker.com/compose/).
 
-### Run ###
+    ```yaml
+    ---
+    version: "3.7"
 
-This Docker image needs a bind mount to get the output from the script to the
-host.
+    services:
+      vdp-scanner:
+        image: 'cisagov/vdp-scanner:0.0.2'
+        volumes:
+          - .:/task/host_mount
+    ```
 
-Using `docker run`
+1. Start the container and detach:
 
-```console
-docker run --mount type=bind,source=$(pwd),target=/task/host_mount --rm cisagov/vdp-scanner
-```
+    ```console
+    docker-compose up --detach
+    ```
+
+## Updating your container ##
+
+### Docker Compose ###
+
+1. Pull the new image from Docker hub:
+
+    ```console
+    docker-compose pull
+    ```
+
+1. Recreate the running container by following the [previous instructions](#running-with-docker-compose):
+
+    ```console
+    docker-compose up --detach
+    ```
+
+### Docker ###
+
+1. Stop the running container:
+
+    ```console
+    docker stop <container_id>
+    ```
+
+1. Pull the new image:
+
+    ```console
+    docker pull cisagov/vdp-scanner:0.0.2
+    ```
+
+1. Recreate and run the container by following the [previous instructions](#running-with-docker).
+
+## Image tags ##
+
+The images of this container are tagged with
+[semantic versions](https://semver.org).  It is recommended that most users use
+a version tag (e.g. `:0.0.2`).
+
+| Image:tag | Description |
+|-----------|-------------|
+|`cisagov/vdp-scanner:1.2.3`| An exact release version. |
+|`cisagov/vdp-scanner:1.2`| The most recent release matching the major and minor version numbers. |
+|`cisagov/vdp-scanner:1`| The most recent release matching the major version number. |
```

Comment:
These version numbers should be changed to agree with the current version of the Docker image.

Reply:
Good thought. Fixed in 2e5b580.